.travis.yml

```diff
@@ -31,6 +39,22 @@ matrix:
   allow_failures:
     - os: osx
 script: |
+  set -e
```

Comment:
Essentially, yes.  I think travis executes the script with `-e` set by default but I like to be explicit.